runtimes/onnx-tf/development/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:18.04
```

Comment:
Why are we using the old Ubuntu version? Let's use 20.04 instead.

Reply:
Done